packages/admin-ui/src/page/page-header-actions.ts

```diff
@@ -0,0 +1,46 @@
+import type { ComponentPropsWithRef } from 'react'
+import { Children } from 'react'
+import { createComponent, useElement } from '@vtex/admin-ui-react'
+
+import * as style from './page.style'
+
+/**
+ * Page header actions component
+ *
+ * @example
+ * import { PageHeader, PageHeaderTop, PageHeaderActions, Button } from "@vtex/admin-ui"
+ *
+ * <PageHeader>
+ *  <PageHeaderTop>
+ *    <PageHeaderTitle>
+ *      Product #123
+ *    </PageHeaderTitle>
+ *    <PageHeaderActions>
+ *      <Button size="large" bleedY>Create</Button>
+ *      <Button size="large" bleedY variant="critical">Delete</Button>
+ *      <MenuButton
+ *        state={state}
+ *        variant="tertiary"
+ *        size="large"
+ *        labelHidden
+ *        bleedY
+ *      />
+ *      <Menu state={state} aria-label="actions">
+ *        <MenuItem label="Create" icon={<IconPlus />} />
+ *        <MenuItem label="Edit" icon={<IconPencil />} />
+ *      </Menu>
+ *    </PageHeaderActions>
+ *  </PageHeaderTop>
+ * </PageHeader>
+ */
+export const PageHeaderActions = createComponent<'div'>((props) => {
+  const { children, ...htmlProps } = props
+
+  return useElement('div', {
+    baseStyle: style.pageHeaderActions,
+    children,
```

Comment:
Avoid the definition of spacing in the children, it's a better practice to always define this in a parent container. In this case you can use the `Stack` component to do it
```suggestion
    children: (
      <Stack space="$l" direction="row">
        {children}
      </Stack>
    ),
```

Reply:
Applied 😉